build-logic/publication/src/main/kotlin/convention.sonatype.gradle.kts

```diff
@@ -1,69 +1,31 @@
-import com.kaspersky.kaspresso.publication.CreateStagingRepositoryTask
-
 plugins {
     id("convention.publication-base")
     signing
 }
 
-val ossrhUsername: Provider<String> = providers.gradleProperty("kaspresso.ossrh.user")
-    .forUseAtConfigurationTime()
-
-val ossrhPassword: Provider<String> = providers.gradleProperty("kaspresso.ossrh.password")
-    .forUseAtConfigurationTime()
-
-val ossrhStagingProfileId: Provider<String> = providers.gradleProperty("kaspresso.ossrh.stagingProfileId")
-    .forUseAtConfigurationTime()
-
-val sonatypeRepoName = "SonatypeReleases"
+val ossrhUsername = findProperty("kaspresso.ossrh.user")?.toString()
```

Comment:
Did you check that configuration cache is not broken here?

Reply:
we don't use configuration cache. No, I'm don't check this point.
I think we check all places when we add configuration cache support